### ISSUE TITLE
[Bug] TypeError: Cannot read property 'id' of undefined (tagged commit with diff-search)

### ISSUE THREAD

**Issue by spinlud** [EndBug/version-check#36]
Hi there!
I have pushed a tagged commit and received the following error:

```
TypeError: Cannot read property 'id' of undefined
```

![image](https://user-images.githubusercontent.com/12783208/104127842-af6e2b80-5364-11eb-9738-00f3394af12a.png)

This is the commit: https://github.com/spinlud/sequelize-typescript-generator/commit/a0198e633f994801425f2b91b7f6c3c143c3cb7f

This is the action configuration:

```yml
      - name: Check version changes
        uses: EndBug/version-check@v1
        id: check
        with:
          diff-search: true

      - name: Publish
        if: steps.check.outputs.changed == 'true' && github.ref == 'refs/heads/master'
        run: |
          npm set registry "https://registry.npmjs.org"
          npm set //registry.npmjs.org/:_authToken ${{ secrets.NPM_PUBLISH_TOKEN }}
          npm publish
```

Any idea how to fix?

**Comment by EndBug** [comment]
Hi, thanks for the issue. 
Just a question: did push only the tag or you pushed some changes along with the tag?
It seems like the action couldn't retrieve the commit list from the payload, but I think that pushing the tag only could have caused the issue.
Anyway, there should be a check that prevents the action from throwing when there are 0 commits.

**Comment by spinlud** [comment]
If I remember correctly, I've pushed the commit and then the tag alone, so probably your guess is right

**Comment by EndBug** [comment]
@all-contributors please add @spinlud for their bug report

**Comment by EndBug** [comment]
The fix has been released in `v2.0.1` (`v2`, `latest`)

Thanks for the issue!
